tasks/pkg-debian.yml

```diff
@@ -3,7 +3,15 @@
   apt: name=apt-transport-https state=present
 
 - name: Install ubuntu apt-key server
-  apt_key: id=A2923DFF56EDA6E76E55E492D3A80E30382E94DE keyserver=hkp://keyserver.ubuntu.com:80 state=present
+  shell: >
+    export KEY=A2923DFF56EDA6E76E55E492D3A80E30382E94DE;
```

Comment:
Not sure if one is better than the other, but I typically sent env vars in ansible via:

```
environment:
         KEY: A2923DFF56EDA6E76E55E492D3A80E30382E94DE
```

Reply:
True, that's probably clearer.